README.md

```diff
@@ -10,6 +10,13 @@ The current releases have been tested on minikube v1.12.3 running k8s v1.18.3
 
 See docs/index.md which is also https://sonatype.github.io/helm3-charts/
 
+### Contributing
+
+See the [contributing document](./CONTRIBUTING.md) for details.
+
+For Sonatypers, note that external contributors must sign the CLA and
+which the Dev-Ex team must verify prior to accepting any PR.
```

Comment:
Language: the word 'which' feels out of place here

Reply:
> 
> 
> Language: the word 'which' feels out of place here

Updated